code/script.js

```diff
@@ -1,9 +1,55 @@
-// A function that adds and remove the class "active" on the section you click on.
-// We haven't really talked about ´this´ yet, but we will... ;)
-// console.log(this) to see how it works
+ 
+document.getElementById("burger").onclick = toggleButton;
+
+document.getElementById("signUp").onsubmit = validateForm;
+
+var questions = document.getElementsByClassName("faq-question");
+for (var i = 0; i < questions.length; i++) {
+  questions[i].onclick = toggle;
+}
+
+var toggleDisplayClass = "toggle-display";
+var toggleElements = document.getElementsByClassName(toggleDisplayClass);
+for (var i = 0; i < toggleElements.length; i++) {
+  var specificElement = toggleElements[i];
+  specificElement.onmouseover = toggleDisplay;
+  specificElement.onmouseout = toggleDisplay;
+}
+
+var listOfVideos = document.getElementsByTagName("video");
+for (var i = 0; i < listOfVideos.length; i++) {
+  var specificVideo = listOfVideos[i]; 
+  specificVideo.onmouseover = playVideo;
+  specificVideo.onmouseout = pauseVideo;
+}
+
+
+function validateForm () {
+  if (document.getElementById("termsandconditions").checked) {
+    return true;
+  } 
+  else {
+    alert('You must read and agree to the Terms & Conditions to sign up for class!'); 
+    return false; 
+  }
+}
+
 function toggle() {
-  this.classList.toggle("active")
+  this.classList.toggle("open");
+}
+      
+function toggleDisplay() {
```

Comment:
Try use the new way of writing functions (ES6) so you know how to write them when you are done at the bootcamp, like this `const toggle = () => {}`